index.js

```diff
@@ -54,6 +54,10 @@ Web3ProviderEngine.prototype.start = function(cb = noop){
         this.emit('error', err)
         return
       }
+      if (!block) {
+        this.emit('error', new Error("Could not find block"))
```

Comment:
Is there a good reason to emit this error event here? It causes my application to exit. I'm not familiar with this event emiting concept, but maybe if we emit a warning it will not exit the application? Also this doesn't seem like an error if the application can transparently recover.